doc/Advanced.md

```diff
@@ -1086,8 +1086,12 @@ The values can be accessed in various ways.
 
 ### Access optional values with or without activity
 
+The value for the key, runType and beamType is returned if found, or an empty value otherwise. 
+However, before returning an empty value we try to substitute the runType and the beamType with "default". 
+
 ```c++
-// returns an Optional<string> if it finds the key `myOwnKey` for the runType and beamType of the provided activity. 
+// returns an Optional<string> if it finds the key `myOwnKey` for the runType and beamType of the provided activity, 
```

Comment:
In the current example I think there is some mixed use of `"myOwnKey"` and `"myOwnKey1"`, my impression is that is there should be `"myOwnKey1"` everywhere.